packages/manager/apps/key-management-service/src/pages/create/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+import {
+  OsdsText,
+  OsdsDivider,
+  OsdsBreadcrumb,
+} from '@ovhcloud/ods-components/react';
+
+import { Notifications } from '@ovhcloud/manager-components';
+import {
+  ODS_THEME_COLOR_INTENT,
+  ODS_THEME_TYPOGRAPHY_SIZE,
+  ODS_THEME_TYPOGRAPHY_LEVEL,
+} from '@ovhcloud/ods-common-theming';
+import { useTranslation } from 'react-i18next';
+import { ROUTES_URLS } from '@/routes/routes.constants';
+import RegionSelector from '@/components/layout-helpers/Create/RegionSelector';
+import OrderConfirmation from '@/components/layout-helpers/Create/OrderConfirmation';
+
+export default function Create() {
+  const { t } = useTranslation('key-management-service/create');
+  const [orderInitiated, setOrderInitiated] = useState(false);
```

Comment:
boolean property should start with `is...`

Reply:
refactored with is..